.github/workflows/pr-build-image.yaml

```diff
@@ -1,17 +1,3 @@
-# Copyright Red Hat, Inc.
```

Comment:
bit weird to remove a copyright statement... but I guess if you have a reason for this...?